source/read-write-concern/tests/operation/default-write-concern-3.2.yml

```diff
@@ -0,0 +1,125 @@
+other_collection_name: &other_collection_name other_collection_name
+
+runOn:
+    - minServerVersion: "3.2"
+
+tests:
+  - description: 'findAndModify operations omit default write concern'
+    clientOptions: {writeConcern: {}}
+    operations:
+      - name: findOneAndUpdate
+        object: collection
+        databaseOptions: {writeConcern: {}}
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          filter: {_id: 1}
+          update: {$set: {x: 1}}
+      - name: findOneAndReplace
+        object: collection
+        databaseOptions: {writeConcern: {}}
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          filter: {_id: 2}
+          replacement: {x: 2}
+      - name: findOneAndDelete
+        object: collection
+        databaseOptions: {writeConcern: {}}
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          filter: {_id: 2}
+    outcome:
+      collection:
+        name: *collection_name
+        data:
+          - {_id: 1, x: 1}
+    expectations:
+      - command_started_event:
+          command:
+            findAndModify: *collection_name
+            query: {_id: 1}
+            update: {$set: {x: 1}}
+            writeConcern: null
+      - command_started_event:
+          command:
+            findAndModify: *collection_name
+            query: {_id: 2}
+            update: {x: 2}
+            writeConcern: null
+      - command_started_event:
+          command:
+            findAndModify: *collection_name
+            query: {_id: 2}
+            remove: true
+            writeConcern: null
+  - description: 'findOneAndUpdate with client write concern unset omits default write concern'
+    operations:
+      - name: findOneAndUpdate
+        object: collection
+        databaseOptions: {writeConcern: {}}
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          filter: {_id: 1}
+          update: {$set: {x: 1}}
+    expectations:
+      - command_started_event:
+          command:
+            findAndModify: *collection_name
+            query: {_id: 1}
+            update: {$set: {x: 1}}
+            writeConcern: null
+  - description: 'findOneAndUpdate with database write concern unset omits default write concern'
+    clientOptions: {writeConcern: {}}
+    operations:
+      - name: findOneAndUpdate
+        object: collection
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          filter: {_id: 1}
+          update: {$set: {x: 1}}
+    expectations:
+      - command_started_event:
+          command:
+            findAndModify: *collection_name
+            query: {_id: 1}
+            update: {$set: {x: 1}}
+            writeConcern: null
+  - description: 'findOneAndUpdate with collection write concern unset omits default write concern'
+    clientOptions: {writeConcern: {}}
+    operations:
+      - name: findOneAndUpdate
+        object: collection
+        databaseOptions: {writeConcern: {}}
```

Comment:
Test title says "collection write concern" but this assigns a database-level WC. Is this backwards with the previous test, which says "database write concern" and sets collection-level?

Reply:
It was intended, as the collectionOptions is the one that is unset. But this is no longer applicable now that it's changed to only set at one level.